addons/web/static/src/core/commands/command_palette.js

```diff
@@ -251,10 +260,15 @@ export class CommandPalette extends Component {
         }
     }
 
-    async executeSelectedCommand() {
+    async executeSelectedCommand(newTab) {
```

Comment:
(answering for the OnCommandClick function comment, idk why but I can't answer to it)

Again from what I read in the tasks' log, it looks like it's there so that one can use "ctrl+click" without triggering the action in the current tab.
However "ctrl(cmd)+click" does not work on my side. I'll take a deeper look with your comment in mind

Reply:
Maybe add a small comment in the code then?